### DISCUSSION TITLE
[Feature] Filter for external library

### DISCUSSION THREAD

**Discussion by alexw-at** [immich-app/immich#14568] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

It would be nice to have search filter options for external libraries.
- Show images not from external libraries
- Show images from one / multiple / all external libraries

### Platform

- [ ] Server
- [X] Web
- [x] Mobile

**Comment by bo0tzz** [comment]
#7587
